From f553a8641cf6b3e0c876e1b85c65d59d4105d0ad Mon Sep 17 00:00:00 2001
From: Hu Kejun <william.hu@rock-chips.com>
Date: Thu, 23 Aug 2018 16:18:10 +0800
Subject: [PATCH] media: rockchip: isp1: change needed min buffers of stream to
 0

change needed min buffers of stream to 0, because we allocate dummy
buffer in advance.

Change-Id: Ib7647983b495c11dc18151b3c1f8856c49496c3a
Signed-off-by: Hu Kejun <william.hu@rock-chips.com>
---
 drivers/media/platform/rockchip/isp1/capture.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/platform/rockchip/isp1/capture.c b/drivers/media/platform/rockchip/isp1/capture.c
index 00d9df43ecc6..5cf6bd7f98d2 100644
--- a/drivers/media/platform/rockchip/isp1/capture.c
+++ b/drivers/media/platform/rockchip/isp1/capture.c
@@ -62,8 +62,8 @@
  * available sp source fmts: yuv, rgb
  */
 
-#define CIF_ISP_REQ_BUFS_MIN 1
-#define CIF_ISP_REQ_BUFS_MAX 8
+#define CIF_ISP_REQ_BUFS_MIN			0
+#define CIF_ISP_REQ_BUFS_MAX			8
 
 #define STREAM_PAD_SINK				0
 #define STREAM_PAD_SOURCE			1
-- 
2.35.3

